Index: src/ex_10082024/Lab014.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>a=10\r\nb=20\r\nprint(a,b, sep=\"()\")
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/ex_10082024/Lab014.py b/src/ex_10082024/Lab014.py
--- a/src/ex_10082024/Lab014.py	(revision 539dcdf49b828135e2657ae549d69ca34c7cd2cd)
+++ b/src/ex_10082024/Lab014.py	(date 1724958482127)
@@ -1,3 +1,3 @@
 a=10
 b=20
-print(a,b, sep="()")
\ No newline at end of file
+print(a,b, sep ="()")
\ No newline at end of file
